src/puppetlabs/puppetdb/utils.clj

```diff
@@ -288,6 +288,16 @@
     (subs s 1 (dec (count s)))
     s))
 
+(defn single-quote
+  [s]
+  (str "'" s "'"))
+
+(defn comma-separated-keywords
+  [words]
+  (->> words
+       (map single-quote)
+       (string/join ", ")))
+
```

Comment:
Not sure it matters, but if we want this to be general purpose, and if we want the output to be unambiguous, we could do something like this instead: `(pr-str (name k))`, which would also use the more natural double-quotes.  e.g. for `(keyword "this is a keyword")`.  Of course that assumes that we want to strip namespaces from keywords.  If not, then we'd want `str` (and perhaps remove the leading colon) instead of `name`.